tendrl/commons/jobs/__init__.py

```diff
@@ -94,11 +95,10 @@ def process_job(jid):
         pass
 
     # tendrl-node-agent tagged as tendrl/monitor will ensure
-    # >10 min old "new" jobs are timed out and marked as
-    # "failed" (the parent job of these jobs will also be
-    # marked as "failed")
-    if "tendrl/monitor" in NS.node_context.tags and \
-        _timeout == "yes" and job.status == "new":
+    # >10 min old "new" parent jobs are timed out and marked 
+    # as "failed"
+    if "tendrl/monitor" in NS.node_context.tags and _timeout == "yes" and \
+        job.status == "new" and job.payload.get('parent') is None:
```

Comment:
timeout only parent job, when it is not processed for a long time.  Because child job will be timed out by a parent job with the proper log message. Timing out from here not give an exact error message.